python/cugraph/centrality/edge_betweenness_centrality_wrapper.pyx

```diff
@@ -0,0 +1,135 @@
+    #       c_vertices = vertices.__cuda_array_interface__['data'][0]
+    if vertices is not None:
+        c_vertices = np.array(vertices, dtype=np.int32).__array_interface__['data'][0]
+
+    c_k = 0
+    if k is not None:
+        c_k = k
+
+    # NOTE: The current implementation only has <int, int, float, float> and
+    #       <int, int, double, double> as explicit template declaration
+    #       The current BFS requires the GraphCSR to be declared
+    #       as <int, int, float> or <int, int double> even if weights is null
+    if result_dtype == np.float32:
+        graph_float = GraphCSRView[int, int, float](<int*> c_offsets,
+                                                    <int*> c_indices,
+                                                    <float*> NULL,
+                                                    number_of_vertices,
+                                                    number_of_edges)
+        # fixme: there might be a way to avoid manually setting the graph property
+        graph_float.prop.directed = type(input_graph) is DiGraph
+
+        c_edge_betweenness_centrality[int, int,
+                                      float, float](graph_float,
+                                                    <float*> c_betweenness,
+                                                    normalized,
+                                                    <float*> c_weight,
+                                                    c_k,
+                                                    <int*> c_vertices)
+        graph_float.get_source_indices(<int*>c_src_identifier)
+    elif result_dtype == np.float64:
+        graph_double = GraphCSRView[int, int, double](<int*>c_offsets,
+                                                      <int*>c_indices,
+                                                      <double*> NULL,
+                                                      number_of_vertices,
+                                                      number_of_edges)
+        # FIXME: there might be a way to avoid manually setting
+        #        the graph property
+        graph_double.prop.directed = type(input_graph) is DiGraph
+
+        c_edge_betweenness_centrality[int, int,
+                                      double, double](graph_double,
+                                                      <double*> c_betweenness,
+                                                      normalized,
+                                                      <double*> c_weight,
+                                                      c_k,
+                                                      <int*> c_vertices)
+        graph_double.get_source_indices(<int*>c_src_identifier)
+    else:
+        raise TypeError("result type for betweenness centrality can only be "
+                        "float or double")
+
+    # FIXME: For large graph renumbering produces a dataframe organized
+    #       in buckets, i.e, if they are 3 buckets
```

Comment:
The output of renumbering is not sorted. coo2csr does temporarily sort the edge list later on to build the adj list.
Is this a problem here?

Reply:
Corrected the comment to `unrenumbering` rather than `renumbering` in 7e7b668.
The order might be counter intuitive if unaware of this behavior, for smaller graphs that are not using the bucket mechanism the output is sorted, while for larger graphs the results is organized by buckets.
Sorting the result is required in the testsuites depending on how we compare the results,
using dictionaries comparison can get slow on large graphs https://github.com/rapidsai/cugraph/pull/876#discussion_r427673671
(taking most of the test time)

Reply:
Got it. This is not sorted because we use a hashing for that feature. Sorting would increase e2e time.